include/ddc/for_each.hpp

```diff
@@ -13,8 +13,8 @@ struct serial_policy
 {
 };
 
-/** iterates over a 1d domain
- * @param[in] domain the 1d domain to iterate
+/** iterates over a nD domain
```

Comment:
I guess we don't need to change that in this PR.

Reply:
true

Reply:
```suggestion
/** iterates over a 1D domain
```